src-graphql/client.js

```diff
@@ -0,0 +1,13 @@
+import GraphQLJSClient from '@shopify/graphql-js-client';
```

Comment:
Seems like the "js" is an artifact of the repo name since we have multiple clients. Once we're in the context of JS I don't think it's needed for an import.